app/scripts/modules/core/src/plugins/plugin.registry.spec.ts

```diff
@@ -53,6 +53,7 @@ describe('PluginRegistry', () => {
       pluginModule = {
         plugin: {
           stages: [({ id: 'mystage' } as any) as IStageTypeConfig],
+          undefinedExtensions: () => {},
```

Comment:
I don't know what `undefinedExtensions` means.  Let's call this `initialize()` ?

Reply:
sure thing! that works for me